Mathlib/Analysis/Distribution/ContDiffMapSupportedIn.lean

```diff
@@ -453,10 +463,13 @@ noncomputable def structureMapCLM (i : ℕ) :
   toLinearMap := structureMapLM 𝕜 n i
   cont := continuous_iInf_dom continuous_induced_dom
 
+-- TODO: Should this be `@[simp]` instead of the one below? I don't want `simp` to
+-- force `WithOrder` variants on people, but maybe this is not a good argument.
```

Comment:
I think it's weird to mark only the `⊤` version as `simp`. I would say either with `withOrder` variant, or neither.